src/dev/build/lib/fs.js

```diff
@@ -70,7 +71,10 @@ export async function copyAll(sourceDir, destination, options = {}) {
   const {
     select = ['**/*'],
     dot = false,
+    atime,
+    mtime,
   } = options;
+  const setTimes = Boolean(atime && mtime);
```

Comment:
Maybe this is a better option, `setTimes: new Date()`. This isn't a general purpose tool anyway so specific options for our needs make the most sense IMO.

Reply:
Rather than specifying `atime` and `mtime` separately that is.